IdentitySdkCore.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |spec|
   spec.homepage              = "https://github.com/ReachFive/identity-ios-sdk"
   spec.license               = { :type => "MIT", :file => "LICENSE" }
   spec.author                = "ReachFive"
-  spec.authors               = { "egor" => "egor@reach5.co", "guillaume" => "guillaume@reach5.co", "roxane" => "roxane@reach5.co", "Pierre" => "pierre.bar@reach5.co", "gbe" => "guillaume.bersac@reach5.co", "Matthieu" => "matthieu@reach5.co" }
+  spec.authors               = { "egor" => "egor@reach5.co", "guillaume" => "guillaume@reach5.co", "roxane" => "roxane@reach5.co", "Pierre" => "pierre.bar@reach5.co", "gbe" => "guillaume.bersac@reach5.co", "Matthieu" => "matthieu@reach5.co", "François" => "francois.devemy@reach5.co" }
```

Comment:
We should probably remove the other authors since they're effectively not reachable anymore.

Reply:
ok